monai/handlers/checkpoint_saver.py

```diff
@@ -91,18 +101,39 @@ def __init__(
         self._final_checkpoint = self._key_metric_checkpoint = self._interval_checkpoint = None
         self._name = name
 
+        class _DiskSaver(DiskSaver):
```

Comment:
I think this can be a workaround until we start a proper support of fixed filename inside ignite Checkpoint.

Reply:
Sounds good, thanks!